person.rb

```diff
@@ -1,14 +1,24 @@
-class Person
+require_relative 'nameable'
+require_relative 'capitalize'
+require_relative 'trimmer'
+require_relative 'base_decorator'
+
+class Person < Nameable
   attr_reader :id
   attr_accessor :name, :age, :parent_permission
 
   def initialize(age, parent_permission: true, name: 'Unknown')
+    super()
     @id = rand(1..1000)
     @name = name
     @age = age
     @parent_permission = parent_permission
   end
 
+  def correct_name
+    @name
```

Comment:
* Kindly fix this error i get when i try to decorate a person `person = Person.new(22, 'maximilianus')
  person.correct_name
  capitalizedPerson = CapitalizeDecorator.new(person)
  capitalizedPerson.correct_name
  capitalizedTrimmedPerson = TrimmerDecorator.new(capitalizedPerson)
  capitalizedTrimmedPerson.correct_name`
![image](https://user-images.githubusercontent.com/52497246/186261908-4f6abe7b-0f8a-4b93-979e-7ef08160a1f8.png)